WakaTime/Watcher.swift

```diff
@@ -236,7 +236,8 @@ class FileMonitor {
     public var eventHandler: (() -> Void)?
 
     init?(filePath: String, queue: DispatchQueue) {
-        fileURL = URL(fileURLWithPath: filePath)
+        guard let fileURL = URL(string: filePath) else { NSLog("No valid path: \(filePath)"); return nil }
```

Comment:
linter failed because `guard` needs an empty line after.

Reply:
I added it